nix-tools/static/zipped.nix

```diff
@@ -0,0 +1,97 @@
+inputs: pkgs:
```

Comment:
[This snippet ](https://github.com/input-output-hk/haskell-nix-example/blob/9b130e2c0128a55ac02820d3b297c07a6ddade10/flake.nix#L743-L771)